dash_docs/chapters/dash_bio/content_module.py

```diff
@@ -579,6 +579,179 @@
+        'url': 'https://hgdownload.cse.ucsc.edu/goldenPath/mm10/bigZips/mm10.2bit'
+    }
+)'''
+        },
+
+        {
+            'param_name': 'Tracks',
+            'description': 'Add tracks in the `tracks` parameter of a Pileup component '
+                           'to view data sources. Tracks can display genomic coverage, features, '
+                           'variants, and reads.'
+                           '\n \n'
+                           'Each track in a Pileup component requires a visualization type (viz) '
+                           'and a data source (source). '
+                           '\n###### Pileup Data Sources\n'
+                           'Each data source requires different parameters. '
+                           'Below, we enumerate the available data source types:'
+                           '\n- `bam`: requires `\'sourceOptions\': {\'url\':, URL.bam \'indexUrl\': URL.bam.bai}`'
+                           '\n- `alignmentJson`: requires `\'sourceOptions\': {GA4GH_JSON_STRING}`'
+                           '\n- `variantJson`: requires `\'sourceOptions\': {GA4GH_JSON_STRING}`'
+                           '\n- `featureJson`: requires `\'sourceOptions\': {GA4GH_JSON_STRING}`'
+                           '\n- `ideogramJson`: requires `\'sourceOptions\': {JSON_STRING}`'
+                           '\n- `vcf`: requires `\'sourceOptions\': {\'url\':, URL.vcf }`'
+                           '\n- `bigBed`: requires `\'sourceOptions\': {\'url\':, URL.bb }`'
+                           '\n'
+                           '\n###### Pileup Visualization Types\n'
+                           'The Pileup component supports the following visualizations:'
+                           '\n- `coverage`: requires `alignmentJson`, `bam`, or `featureJson` source'
+                           '\n- `genes`: requires `bigBed` source'
+                           '\n- `features`: requires `featureJson` or `bigBed` source'
+                           '\n- `variants`: requires a `vcf` or `variantJson` data source'
+                           '\n- `genotypes`: requires a `vcf` or `variantJson` data source'
+                           '\n- `pileup`: requires a `bam` or `alignmentJson` data source'
+                           '\n- `idiogram`:  requires `ideogramJson` data source'
+                           '\n- `location`: does not require a data source'
+                           '\n- `scale`: does not require a data source'
+                           '\n \n'
+                           'Multiple tracks can be added to a Pileup component by creating a list '
+                           'of dictionaries, each of which corresponds to a single track.',
+            'code': '''import dash_bio as dashbio
+import os
+import json
+import six.moves.urllib.request as urlreq
+
+# read in JSON
+json = urlreq.urlopen('https://raw.githubusercontent.com/plotly/dash-bio-docs-files/master/pileup.synth4.tumor.chr1.4930000-4950000.json').read()
```

Comment:
```suggestion
source_data = urlreq.urlopen('https://raw.githubusercontent.com/plotly/dash-bio-docs-files/master/pileup.synth4.tumor.chr1.4930000-4950000.json').read()
```
I think we want to avoid naming variables the same as common modules, it will prevent confusion and unintended interactions with other modules (eg. `pickle`).